commons/tools/crons.md

```diff
@@ -46,46 +46,53 @@ Example of `clevercloud/cron.json` which executes the file `cron.php` every 5 mi
 
 ## Access environment variables
 
-Environment variables are not available in scripts / executables ran from crons. You must inject them yourself.
+To access environment variable, you must wrap your commands in a bash script. Let's say
```

Comment:
I would write "To have access to environment variables" (in any case, the 's' of variables is missing)